src/modules/roc_netio/target_uv/roc_netio/udp_receiver_port.cpp

```diff
@@ -263,5 +275,29 @@ void UDPReceiverPort::recv_cb_(uv_udp_t* handle,
     self.writer_.write(pp);
 }
 
+bool UDPReceiverPort::join_multicast_group_() {
+    if (int err = uv_udp_set_membership(&handle_, packet::ip_to_str(address_).c_str(),
```

Comment:
I suggest to add a debug message that we're joining/leaving group - it may be an important event for the user.

Reply:
Done